duckduckgo/iOS
# extend valid urls to include IPv6

Task/Issue URL: https://app.asana.com/0/1177771139624306/1203127427026896/f

**Description**: Makes IPv6 valid url when used in navigation.



**Steps to test this PR**:
1. Type on the search bar http://[::]:8080/ it should navigate to it rather than searching
2. Try the same for other IPv6 or IPv4 addresses ex:
http://192.168.0.1/
http://203.0.113.0/
http://[2001:0db8:85a3:0000:0000:8a2e:0370:7334]
http://[2001:0db8::1]



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
